pandas/tests/series/test_constructors.py

```diff
@@ -1810,6 +1810,20 @@ def test_constructor_bool_dtype_missing_values(self):
         expected = Series(True, index=[0], dtype="bool")
         tm.assert_series_equal(result, expected)
 
+    @pytest.mark.parametrize("any_int_dtype", ["int64"])
+    def test_constructor_int64_dtype(self, any_int_dtype):
```

Comment:
no, pls just use the fixture itself, e.g. no parameterize

Reply:
This is causing Assertion Error.

Reply:
![image](https://user-images.githubusercontent.com/63910248/147833002-384bb60d-c65d-469c-b43e-e70361969744.png)

Reply:
The previous code segment is leading to this issue, if we have only int64 there is no issue.

Reply:
you need to match the expected value as well

Reply:
I have added them in the parameters which are passing.

uint -> uint8, uint16, uint32, uint64 are failing due to internal code implementation
Float -> Float32, Float64 are failing as it is unable to implicitly cast strings.

The other have been added into the parametrize function defined, which are not causing any problem.

Reply:
u can xfail the failing ones that are tricky to fix eg Float but the others should work

Reply:
![image](https://user-images.githubusercontent.com/63910248/147844312-6ecc6742-086a-4e03-8b56-07c239fc7dae.png)

Reply:
I am getting this for uint.

Reply:
@jreback is this fine?

Reply:
@jreback I think I have covered everything?

Reply:
@shubham11941140 you are not using the fixtures pls do so

Reply:
just remove the paramterize completely

Reply:
uint -> uint8, uint16, uint32, uint64 are failing due to internal code implementation. Do i fix this?

Reply:
@jreback removed parametrization, now it should be ready.